inc/query/query.php

```diff
@@ -152,4 +146,50 @@ public function handle_query_sibling( &$clauses, $id_column ) {
 
 		$clauses['where'] .= empty( $clauses['where'] ) ? $where : " AND $where";
 	}
+
+	/**
+	 * Modify query join & where statement for multi-relationship.
+	 *
+	 * @param string $clauses   Query clauses.
+	 * @param array  $args   $WP_query args object.
+	 */
+	public function handle_multiple_relationships( &$clauses, $id_column ) {
+		global $wpdb;
+		$this->relation = $this->args['relation'];
```

Comment:
Biến `$this->relation` này được dùng mỗi trong hàm này, nên a nghĩ ko cần tạo thành prop của class mà chỉ cần để là biến bình thường trong hàm là đc.

Reply:
đã fix cám ơn anh.